flake.nix

```diff
@@ -0,0 +1,63 @@
+{
```

Comment:
This is for our dev shell and CI.  Anyone who doesn't know what this is can ignore it.